pimpmycause/pimpmycause/templates/registration/password_change_form.html

```diff
@@ -0,0 +1,16 @@
+{% extends "layout.html" %}
+
+{% block title %} Change Password {% endblock %}
+
+{% load static %}
+
+{% block main %}
+
+    <h1>Change Password</h1>
+
+    <form method="POST" action="" class="row">
+        {% csrf_token %}
+        {{ form.as_p }}
+        <button type="submit" name="action">Change Password</button>
```

Comment:
Please add the button classes `button` and `button--primary` for pink colour, same as above please wrap everything similar to other pages so they all match